src/Xamarin.Android.Build.Tasks/Tasks/JavaToolTask.cs

```diff
@@ -66,8 +66,8 @@ at com.android.dx.command.dexer.Main.main(Main.java:215)
 			at com.android.dx.command.Main.main(Main.java:106)
 		*/
 		const string ExceptionRegExString = @"(?<exception>java.lang.+):(?<error>.+)";
-		Regex codeErrorRegEx = new Regex (CodeErrorRegExString, RegexOptions.Compiled);
-		Regex exceptionRegEx = new Regex (ExceptionRegExString, RegexOptions.Compiled);
+		protected Regex codeErrorRegEx = new Regex (CodeErrorRegExString, RegexOptions.Compiled);
```

Comment:
This should probably be `protected static readonly`. This isn't something that should be changed (`readonly`), and I can't think of any reason we'd ever need to change this on a per-instance basis (`static`).